README.md

```diff
@@ -115,13 +115,15 @@ _or_ use the mirror at the [University of Illinois Urbana-Champaign](https://ill
 
 (In either example, replace `focal` with `jammy` for use with Ubuntu 22.04.)
 
-Third, and optionally, if you do not yet have the current R version, run these two lines (or
+Third, and optionally, if you do not yet have the current R version, run these four lines (or
 use the [standard CRAN repo setup](https://cloud.r-project.org/bin/linux/ubuntu/))
 
     wget -q -O- https://cloud.r-project.org/bin/linux/ubuntu/marutter_pubkey.asc \
         | tee -a /etc/apt/trusted.gpg.d/cran_ubuntu_key.asc
     echo "deb [arch=amd64] https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/" \
         > /etc/apt/sources.list.d/cran-ubuntu.list
+    apt update
+    apt install -y r-base-core
```

Comment:
No I think I prefer the README.md as it is.

Thanks for the suggestion but it is generally better to first _discuss_ a proposal as an issue ticket.